crates/api_common/src/utils.rs

```diff
@@ -1037,7 +1031,7 @@ pub fn check_conflicting_like_filters(
 
 pub async fn process_markdown(
   text: &str,
-  slur_regex: &Option<LemmyResult<Regex>>,
+  slur_regex: &Regex,
```

Comment:
Now that this is no longer an option, isn't it going to throw the new lemmyerror you created?

Reply:
The error would only be thrown if the regex is invalid, and that shouldnt happen as its checked before create/update.

`Option<Regex>` also works, but this way the code is a bit simpler.